java/client/src/test/java/glide/api/RedisClusterClientTest.java

```diff
@@ -96,8 +124,134 @@ public TestCommandManager(Response responseToReturn) {
+
+        // exercise
+        CompletableFuture<String> response = service.ping(message, route);
+        String pong = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(message, pong);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_returns_string() {
+        // setup
+        CompletableFuture<ClusterValue<String>> testResponse = mock(CompletableFuture.class);
+        Map<String, String> testPayload = new HashMap<String, String>();
+        testPayload.put("addr1", "value1");
+        testPayload.put("addr2", "value2");
+        testPayload.put("addr3", "value3");
+        when(testResponse.get()).thenReturn(ClusterValue.of(testPayload));
+        when(commandManager.<ClusterValue<String>>submitNewCommand(
+                        eq(Info), eq(new String[0]), eq(Optional.empty()), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<ClusterValue<String>> response = service.info();
+
+        // verify
+        ClusterValue<String> clusterValue = response.get();
+        assertTrue(clusterValue.hasMultiData());
+        Map<String, String> payload = clusterValue.getMultiValue();
+        assertEquals(testPayload, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_with_route_returns_string() {
+        // setup
+        CompletableFuture<ClusterValue<String>> testResponse = mock(CompletableFuture.class);
+        Map<String, String> testClusterValue = Map.of("addr1", "addr1 result", "addr2", "addr2 result");
+        RequestRoutingConfiguration.Route route = RequestRoutingConfiguration.SimpleRoute.ALL_NODES;
+        when(testResponse.get()).thenReturn(ClusterValue.of(testClusterValue));
+        when(commandManager.<ClusterValue<String>>submitNewCommand(
+                        eq(Info), eq(new String[0]), eq(Optional.of(route)), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<ClusterValue<String>> response = service.info(route);
+
+        // verify
+        ClusterValue<String> clusterValue = response.get();
+        assertTrue(clusterValue.hasMultiData());
+        Map<String, String> clusterMap = clusterValue.getMultiValue();
+        assertEquals("addr1 result", clusterMap.get("addr1"));
+        assertEquals("addr2 result", clusterMap.get("addr2"));
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_with_route_with_infoOptions_returns_string() {
+        // setup
+        String[] infoArguments = new String[] {"ALL", "DEFAULT"};
+        CompletableFuture<ClusterValue<String>> testResponse = mock(CompletableFuture.class);
+        Map<String, String> testClusterValue = Map.of("addr1", "addr1 result", "addr2", "addr2 result");
+        when(testResponse.get()).thenReturn(ClusterValue.of(testClusterValue));
+        RequestRoutingConfiguration.Route route = RequestRoutingConfiguration.SimpleRoute.ALL_PRIMARIES;
```

Comment:
If you fix imports you can do
```suggestion
        Route route = SimpleRoute.ALL_PRIMARIES;
```

Reply:
Cleaned up.